### PR TITLE
With inbox sdk

### PR DESCRIPTION
**Pull request by Aarush-wadhia**
This example includes `with-inbox-sdk`

### CODE REVIEW

**Comment by louisgv** [comment]
@Aarush-wadhia Can you rebase and remove files that doesn't belong to the inbox-sdk

File: with-inbox-sdk/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "typescript": "4.7.4"
+  },
+  "manifest": {
+    "host_permissions": [
+      "https://*/*"
+    ],
+    "permissions": [
+      "scripting"
+    ],
+    "content_scripts": [
```
Comment (louisgv) [inline with-inbox-sdk/package.json:34]: We don't need this, use host_permissions instead for the matches.

In term of content-script, uses: 

https://docs.plasmo.com/workflows/content-scripts#injecting-ui-elements

File: with-jotai/.github/workflows/submit.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: "Submit to Web Store"
```
Comment (louisgv) [inline with-jotai/.github/workflows/submit.yml:1]: remove this file

File: with-inbox-sdk/popup.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { useEffect } from "react"
+import * as InboxSDK from '@inboxsdk/core';
+
+function IndexPopup() {
+  useEffect(() => {
+    InboxSDK.load(2, "YOUR_APP_ID", {}).then((sdk) => {
```
Comment (louisgv) [inline with-inbox-sdk/popup.tsx:6]: use env variable instead

File: with-inbox-sdk/popup.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import * as InboxSDK from '@inboxsdk/core';
+
+function IndexPopup() {
+  useEffect(() => {
+    InboxSDK.load(2, "YOUR_APP_ID", {}).then((sdk) => {
+      sdk.Compose.registerComposeViewHandler((composeView) => {
+        composeView.addButton({
+          title: "My Nifty Button!",
+          iconUrl:
+            "https://lh5.googleusercontent.com/itq66nh65lfCick8cJ-OPuqZ8OUDTIxjCc25dkc4WUT1JG8XG3z6-eboCu63_uDXSqMnLRdlvQ=s128-h128-e365",
```
Comment (louisgv) [inline with-inbox-sdk/popup.tsx:11]: Use this image instead: https://www.plasmo.com/logo.png

File: with-inbox-sdk/package.json (hunk 1 above)
Comment (Aarush-wadhia) [reply with-inbox-sdk/package.json:34]: Okay let me take care of this part

File: with-jotai/.github/workflows/submit.yml (hunk 2 above)
Comment (Aarush-wadhia) [reply with-jotai/.github/workflows/submit.yml:1]: Sure

File: with-inbox-sdk/popup.tsx (hunk 3 above)
Comment (Aarush-wadhia) [reply with-inbox-sdk/popup.tsx:6]: Roger that

File: with-inbox-sdk/popup.tsx (hunk 4 above)
Comment (Aarush-wadhia) [reply with-inbox-sdk/popup.tsx:11]: I will do that

**Pull request closed without merging**